### PR TITLE
refactored js version1

### PR DESCRIPTION
**Pull request by ShreyaPrasad1209**
#### Describe the changes you have made in this PR - 

1. Refactored the JavaScript code in three files under simulator folder as of now. If this PR is approved then I will refactor rest of the JS code.
2. Used ES6 arrow notation and replaced var with let.
3. Used a `for-of` loop instead of a `for` loop.

### Screenshots of the changes (If any) -

None

Note: Please check **Allow edits from maintainers.** if you would like us to assist in the PR.

### CODE REVIEW

File: simulator/src/canvas2svg.js [hunk 1]
```diff
@@ -18,20 +18,20 @@
```
Comment (tachyons) [inline simulator/src/canvas2svg.js:18]: This is a thirdparty lib, better not to change this file

Comment (ShreyaPrasad1209) [reply simulator/src/canvas2svg.js:18]: Okay which files should I not modify under simulator ?

Comment (tachyons) [reply simulator/src/canvas2svg.js:18]: Everything except thirdparty scripts. there are only few such files in simulator folder, rest are moved to npm

Comment (ShreyaPrasad1209) [reply simulator/src/canvas2svg.js:18]: Ok then I am closing this PR.

**Pull request closed without merging**